app/templates/components/forms/login-form.hbs

```diff
@@ -19,7 +19,7 @@
           <div class="field">
             <div class="ui left icon input">
               <i class="mail icon"></i>
-              {{input name='email' placeholder=(t 'Email ID') value=identification}}
+              {{input name='email' placeholder=(t 'Email ID') value=username}}
```

Comment:
This should be identification only as the JS property has identification. That's why the test is failing

Reply:
The issue is due to the error message, Test case expects `Invalid Credentials` whereas the actual is `unexpected error`, Most probably e.errors is being read from server and before that FE is throwing unexpected error

Reply:
That's wrong. Unexpected error is coming because frontend is sending wrong body